packages/grid/src/vaadin-grid-sorter-mixin.js

```diff
@@ -117,6 +117,11 @@ export const GridSorterMixin = (superClass) =>
     /** @protected */
     connectedCallback() {
       super.connectedCallback();
+
+      if (this.performUpdate) {
```

Comment:
To invoke the _pathOrDirectionChanged observer before applySorters.

Reply:
It would be good to add a comment about this.

Reply:
A comment is added.